### PR TITLE
fix bug in yolox_openvino.cpp

### PR DESCRIPTION
**Pull request by amazingzby**
https://github.com/Megvii-BaseDetection/YOLOX/issues/532

### CODE REVIEW

File: demo/OpenVINO/cpp/yolox_openvino.cpp [hunk 1]
```diff
@@ -41,6 +42,8 @@ void blobFromImage(cv::Mat& img, Blob::Ptr& blob){
     int channels = 3;
     int img_h = img.rows;
     int img_w = img.cols;
+    std::vector<float> mean = {0.485, 0.456, 0.406};
+    std::vector<float> std = {0.229, 0.224, 0.225};
```
Comment (ruinmessi) [inline demo/OpenVINO/cpp/yolox_openvino.cpp:46]: We remove the normalization in the new version, so you may have to pull our latest version

Comment (amazingzby) [reply demo/OpenVINO/cpp/yolox_openvino.cpp]: I have removed the normalization in my code. Thank you.

**Review by ruinmessi — COMMENTED** [review]
Have you tested the code in your local env?

File: demo/OpenVINO/cpp/yolox_openvino.cpp [hunk 2]
```diff
@@ -234,7 +236,7 @@ static void decode_outputs(const float* prob, std::vector<Object>& objects, floa
         std::vector<int> strides = {8, 16, 32};
         std::vector<GridAndStride> grid_strides;
 
-        generate_grids_and_stride(INPUT_W, strides, grid_strides);
+        generate_grids_and_stride(INPUT_W, INPUT_H, strides, grid_strides);
```
Comment (ruinmessi) [inline demo/OpenVINO/cpp/yolox_openvino.cpp:239]: This line is inconsistent with the definition of generate_grids_and_stride, which takes "input_h, input_w" order.

**Review by ruinmessi — CHANGES_REQUESTED** [review]
The generate_grids_and_stride function is not correctly used.

File: demo/OpenVINO/cpp/yolox_openvino.cpp (hunk 2 above)
Comment (amazingzby) [reply demo/OpenVINO/cpp/yolox_openvino.cpp:239]: @ruinmessi I have changed input_h and input_w in the function "generate_grids_and_stride" and tested the new submition.

**Review by ruinmessi — COMMENTED** [review]
LGTM.

**Pull request merged**